dumper/app/src/main/resources/oracle-stats/statspack/sql-stats-statspack.sql

```diff
@@ -18,6 +18,9 @@ SELECT
   A.instance_number "InstanceNumber",
   to_char(B.force_matching_signature) "ForceMatchingSignature",
   min(B.sql_id) "SqlId",
+  min(A.snap_time) "MinSnapTime",
```

Comment:
Can we use `MinBeginTime` and `MaxEndTime` names for these columns as they are already used in AWR and assessment ingestion?

Reply:
Done